include/unifex/linux/io_uring_context.hpp

```diff
@@ -42,7 +44,7 @@
 #include <sys/uio.h>
 
 namespace unifex {
-namespace linux {
+namespace linuxos {
```

Comment:
I was debating what to call this namespace.
I think linuxos is a reasonable tradeoff. 👍 

Do we change the directory name to match, though?

Reply:
I don't think so - at least for this diff.